scripts/exgfs_atmos_gempak_gif_ncdc_skew_t.sh

```diff
@@ -7,107 +7,79 @@
 # in the future, we should move it above somewhere else.
 ##############################################################
 
-source "$HOMEgfs/ush/preamble.sh"
+source "${HOMEgfs}/ush/preamble.sh"
 
-cd $DATA
+cd "${DATA}" || exit 1
 
-export NTS=${HOMEgfs}/gempak/ush/restore
+export NTS="${HOMEgfs}/gempak/ush/restore"
 
-if [ $MODEL = GDAS -o $MODEL = GFS ]
-then
-    case $MODEL in
-      GDAS) fcsthrs="00";;
-      GFS)  fcsthrs="00 12 24 36 48";;
+if [[ "${MODEL}" == GDAS ]] || [[ "${MODEL}" == GFS ]]; then
+    case "${MODEL}" in
+        GDAS) fcsthrs="0";;
+        GFS)  fcsthrs="0 12 24 36 48";;
+        *)
+            echo "FATAL ERROR: Unrecognized model type ${MODEL}"
+            exit 5
+            ;;
     esac
 
-    export fhr
-    for fhr in $fcsthrs
-    do
-        icnt=1
-        maxtries=180
-        export GRIBFILE=${COMIN}/${RUN}_${PDY}${cyc}f0${fhr}
-        while [ $icnt -lt 1000 ]
-        do
-          if [ -r ${COMIN}/${RUN}_${PDY}${cyc}f0${fhr} ] ; then
-            sleep 5
-            break
-          else
-            echo "The process is waiting ... ${GRIBFILE} file to proceed."
-            sleep 20
-            let "icnt=icnt+1"
-          fi
-          if [ $icnt -ge $maxtries ]
-          then
-            echo "ABORTING: after 1 hour of waiting for ${GRIBFILE} file at F$fhr to end."
-            export err=7 ; err_chk
-            exit $err
-          fi
-        done
-
-       cp ${COMIN}/${RUN}_${PDY}${cyc}f0${fhr} gem_grids${fhr}.gem
-
-#       if [ $cyc -eq 00 -o $cyc -eq 12 ]
-       #then
-          ${HOMEgfs}/gempak/ush/gempak_${RUN}_f${fhr}_gif.sh
-       #fi
-
+    sleep_interval=20
+    max_tries=180
+    for fhr in ${fcsthrs}; do
+        fhr3=$(printf %03i "${fhr}")
```

Comment:
Should this be `fhr3=$(printf "%03d" "${fhr}")`?

Reply:
They're function almost identically, but fixed.